src/services/decoders/JsonlDecoder/index.ts

```diff
@@ -62,6 +62,15 @@ class JsonlDecoder implements Decoder {
         this.#formatter = new YscopeFormatter({formatString: decoderOptions.formatString});
     }
 
+    static async create (dataArray: Uint8Array, decoderOptions: DecoderOptions) {
+        // eslint-disable-next-line no-magic-numbers
+        if (0 < dataArray.length && 0x7b !== dataArray[0]) {
```

Comment:
is it better to replace `0x7b` with `"{".charCodeAt(0)`? if so, let's also remove the one in the definition object